### PR TITLE
ES7 Support

### PR DESCRIPTION
**Pull request by varunsharma27**
[INVE-36](https://sirensolutions.atlassian.net/browse/INVE-36)
- Adds getTotalHitCount to Migration class

### CODE REVIEW

**Comment by jccq** [comment]
\o/

**Comment by varunsharma27** [comment]
Will release `1.16.0` after the merge.

File: src/migrations/migration.js [hunk 1]
```diff
@@ -31,6 +31,19 @@ export default class Migration {
...
   }
 
+  /**
+   * Retrieves the total hit count from a response object
+   * @param  {Object} response
+   * @return {Number} Total hit count
+   */
+  getTotalHitCount(response) {
+    const total = response.hits.total;
+    if (typeof total === "object") {
```
Comment (varunsharma27) [inline src/migrations/migration.js]: Can also do `if (total && typeof total === "object")` but total is never null.

File: lib/migrations/__tests__/migration.js [hunk 2]
```diff
@@ -40,7 +59,6 @@ describe('migrations', function () {
         return new _migration2.default(configuration);
       }).not.to.throwError();
     });
-
     describe('countHits', function () {
```
Comment (varunsharma27) [inline lib/migrations/__tests__/migration.js]: No tests for count required as count api remains the same: https://github.com/elastic/elasticsearch-js-legacy/blob/128ba27f1ab6609034f7400df4f98e059c2b27c9/docs/api_methods_7_4.asciidoc#count

File: src/migrations/migration.js (hunk 1 above)
Comment (szydan) [inline src/migrations/migration.js]: single quotes please

**Comment by varunsharma27** [comment]
Will delete the branch later, it's being referenced as a dependency in ES7 branches.

File: src/migrations/migration.js (hunk 1 above)
Comment (varunsharma27) [reply src/migrations/migration.js]: Let me open a new PR.

**Pull request merged**